cmd/prometheus/main.go

```diff
@@ -667,6 +670,12 @@ func main() {
 
 			cfg.tsdb.MaxBlockDuration = maxBlockDuration
 		}
+
+		// Delayed compaction checks
+		if cfg.tsdb.EnableDelayedCompaction && (cfg.tsdb.CompactionDelayMaxPercent >= 100 || cfg.tsdb.CompactionDelayMaxPercent < 0) {
```

Comment:
still don't get why we cannot set it to 100, does it break sth?

Reply:
ah no sorry 100 is fine, will update